doczrc.js

```diff
@@ -7,7 +7,6 @@ export default {
     title: 'Lime Elements',
     description: 'Documentation for Lime Elements',
     typescript: true,
-    indexHtml: 'src/index.html',
```

Comment:
The index.html file is no longer used. Which is a bit of a problem, since this is how we loaded lime-elements. So we'll have to load it in some other way. And we'll also have to get Gatsby to actually serve it, because it isn't at the moment. Putting it in the `static` folder instead of the `public` folder might help though: https://www.gatsbyjs.org/docs/static-folder/